### DISCUSSION TITLE
How to find a label with some special characters

### DISCUSSION THREAD

**Discussion by SamuelPinho** [cypress-io/cypress#24883] [Component Testing]
I have an Input field that can have a label with any characters.

An example would be: `This is a custom label ~!@#$%^&*()_+=-[]{}\|:",./? UID`.

I'm trying to find this label by doing something like this:

```jsx
const customLabel = 'This is a custom label ~!@#$%^&*()_+=-[]{}\|:",./? UID'
cy.contains('label', customLabel);
```

But this code is unable to locate the label giving me the following error:
<img width="527" alt="image" src="https://user-images.githubusercontent.com/20033265/204627673-300cdbfc-57ca-4e68-a03c-a85af27ffbdd.png">

**What I'm trying to find:** 'This is a custom label ~!@#$%^&*()_+=-[]{}\|:",./? UID'
**What Cypress is trying to find**: 'This is a custom label ~!@#$%^&()_+='-[']{}\|:",./?   UID'

They both different and I'm not sure why

**Comment by jjhelguero** [comment]
Are you sure there is text containing that string or is it the value?

**Reply by SamuelPinho** [reply]
Yes, here's the screenshot:
<img width="416" alt="image" src="https://user-images.githubusercontent.com/20033265/204926843-3b66a22f-c61f-4f45-90cb-9ded4f79e2a8.png">

And it's weird that the error that Cypress is giving me is that it says that it's a unrecognized expression and not that it didn't find the element on the screen

**Reply by jjhelguero** [reply]
Hmm, I might be reading the error wrong. Have you tried using regexp with escaped characters?

**Reply by SamuelPinho** [reply]
Do you mean, passing a regexp with escaped characters on the `contains` function?
